src/Widgets/ColumnContainerWidget/ColumnsEditorTab.scss

```diff
@@ -342,6 +346,83 @@
   text-shadow: 0 1px 2px #fff;
 }
 
+// -- small screen adaptions -- //
+
+.gle {
+  container-type: inline-size;
+}
+
+@container (max-inline-size: 400px) {
```

Comment:
With this the underlaying grid looks a bit off. E.g. one dragging the handle away without letting loos  for the "big" version it looks good:
<img width="1014" height="974" alt="Moving handle big" src="https://github.com/user-attachments/assets/4d4cb994-e6be-4b7a-9436-2ff7f4d70b37" />

But for the small version it looks off:
<img width="804" height="1037" alt="Moving handle small" src="https://github.com/user-attachments/assets/bbc71468-3c72-43c9-8b71-186cf5101419" />